integration/storage/storage.go

```diff
@@ -44,10 +44,15 @@ func InitCRDStorage(ctx context.Context, h *framework.Host, l micrologger.Logger
 	c.K8sClient = h.K8sClient()
 	c.Logger = l
 
+	targetNamespace := h.TargetNamespace()
+	if targetNamespace == "" {
+		targetNamespace = giantswarmNamespace
+	}
+
 	c.Name = "kvm-e2e"
 	c.Namespace = &v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: giantswarmNamespace,
+			Name: targetNamespace,
```

Comment:
this is wrong, the storage has to be in  same namespace, if you separate storage by namesapce then each test will have its own storage and the whole rangepool magic wont work

Reply:
Shit. You are totally right. We already discussed this. I reused the same branch so I must have committed it before cleaning it out (I did `git checkout..` for changes before working on retry).... Oh well, I'll remove this immediately. Thanks!